.circleci/config.yml

```diff
@@ -0,0 +1,541 @@
+version: 2.1
+
+defaults: &defaults
+  docker:
+    - image: node:6
```

Comment:
pretty sure we move to 8 by default now? I think we can ditch 8 here

Reply:
@lili2311 CLI still supports Node 6. Your team hasn't dropped this yet, we discussed this recently :smile:

Reply:
ah derp this is the plugin ofc

Reply:
there is a plan in motion

Reply:
Great!